en/index.md

```diff
@@ -523,6 +523,44 @@ Lastly, these Latin words and phrases are widespread in academia and research. T
 - stet
 
 
+<!-- RULE
+#27 Active voice and passive voice
+-->
+
+## Active and passive voice
+
+Documentation contains a mix of active and passive voice, and you should use both with care and attention.
+
+Active voice makes the agent – the thing taking action – the subject of the sentence. In "The server hosts all the files", _the server_ is the agent. 
+
+Passive voice occurs when you demote the agent. It either recasts the agent into an adjunct role, such as in "The files are hosted by the server", or omits the agent entirely, as in "The files are hosted".
+
+Whether to use active and passive voice depends on the focus of the sentence. When writing about software, we often describe user actions (_You can do X_) and causes and effects (_A leads to B_). Misusing passive voice tends to disengage the reader or make the software's workings seem inscrutable. Excessive passive voice, where the agent is rarely mentioned across large passages of text, makes for exhausting and tedious reading.
+
+Consider the sentence "In Ubuntu 23.10, apps **are now installed** with the App Center". This would be poor phrasing in a user guide, as the user isn't the focus. We ought to communicate that _they_ should take this action, not some unspecified entity. We can improve this sentence by adding the user in the agent role – "In Ubuntu 23.10, **you can now install** apps with the App Center".
```

Comment:
Personal nitpick perhaps, but I would remove "now" from the examples.
This is unrelated to your core argument but I think we shouldn't normalise using terms that will ultimately date a technical document ("now", "recently", etc.); in fact, that could be a nice entry in the style guide!

Reply:
I was presupposing a release note, but if the context wasn't clear to you then that's a good reason for me to change it.

Reply:
09a29e624e78488be2ce25bdd17c26d16edcd353